angular_components/lib/material_input/material_auto_suggest_input.md

```diff
@@ -0,0 +1,97 @@
+# Overview
```

Comment:
Should this file be named README.md? (That is how the other documentation files are named for the other components.)

Reply:
I was thinking the same thing at first, but this file only discusses the auto suggest input and none of the other components in the directory so I figured we could leave the name as is for now.